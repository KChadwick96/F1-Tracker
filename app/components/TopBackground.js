import React from 'react';
import { View, StyleSheet } from 'react-native';
import PropTypes from 'prop-types';

const TopBackground = ({height}) => {
    const containerStyles = [styles.container, {height}];

    return (
        <View style={containerStyles} />
    )
}

TopBackground.propTypes = {
    height: PropTypes.number.isRequired
};

const styles = StyleSheet.create({
    container: {
        backgroundColor: 'red',
        position: 'absolute',
        top: 0,
        left: 0,
        right: 0,
        height: 200
    }
});

export default TopBackground;