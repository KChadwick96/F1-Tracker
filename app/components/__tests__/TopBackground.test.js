import React from 'react';
import { shallow } from 'enzyme';

import TopBackground from '../TopBackground';

const component = <TopBackground height={100} />;

it('renders correctly', () => {
    const wrapper = shallow(component);
    expect(wrapper).toMatchSnapshot();
});