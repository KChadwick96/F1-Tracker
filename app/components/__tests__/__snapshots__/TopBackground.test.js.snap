// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "red",
      "height": 100,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
/>
`;
